tests/qml/Properties.qml.json

```diff
@@ -5,13 +5,13 @@
       "qmlimport",
       "QtQuick",
       2.5,
-      "",
+      "QQ",
       true
     ]
   ],
   [
     "qmlelem",
-    "Rectangle",
+    "QQ.Rectangle",
```

Comment:
This has changed and will be `["dot",  "QQ", "Rectangle"],` after a rebase.
Note that this line shouldn't even be present here, as `Rectangle` → `QQ.Rectangle` is an unrelated change on this line.